raiden/tests/integration/test_raidenservice.py

```diff
@@ -46,6 +46,7 @@ def test_regression_filters_must_be_installed_from_confirmed_block(raiden_networ
     })
 
 
+@pytest.mark.skip('flaky')
```

Comment:
Can you link to the issue? https://github.com/raiden-network/raiden/issues/3714

Reply:
also it might be better to mark it `xfail`, so it will be run and reported as `xpass` or `xfail`, not just silently skipped